src/Microsoft.ML.Data/Scorers/BinaryClassifierScorer.cs

```diff
@@ -94,7 +94,7 @@ private static bool CanWrap(ISchemaBoundMapper mapper, ColumnType labelNameType)
             if (rowMapper == null)
                 return false; // We could cover this case, but it is of no practical worth as far as I see, so I decline to do so.
 
-            ISchema outSchema = mapper.Schema;
+            ISchema outSchema = mapper.OutputSchema;
```

Comment:
>ISchema [](start = 12, length = 7)

can we just use var here?
I don't see any reason to downgrade Schema to ISchema

Reply:
or maybe just use mapper.OutputSchema directly and don't create this variable at all?

---
In reply to: [237955026](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237955026) [](ancestors = 237955026)

Reply:
I implemented your 2nd suggestion.

---
In reply to: [237955424](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237955424) [](ancestors = 237955424,237955026)